api/v1/helpers/verbs/putUtils.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * api/v1/helpers/verbs/putUtils.js
+ */
+'use strict'; // eslint-disable-line strict
+
+const u = require('./utils');
+const publisher = u.publisher;
+const event = u.realtimeEvents;
+const httpStatus = require('../../constants').httpStatus;
+const redisCache = require('../../../../cache/redisCache').client.cache;
+
+/**
+ * Generic helper to update a resource.
+ * If no value was provided for an field, clears that field by setting its
+ * value to null (or false for boolean fields).
+ *
+ * @param {Object} req From Express
+ * @param {Object} props From Express
```

Comment:
"props" is not from express. It is not the model on which the update operation is performed.